### DISCUSSION TITLE
BTCPayserver support to networks such as i2p?

### DISCUSSION THREAD

**Discussion by decentralizedb** [btcpayserver/btcpayserver#4505] [Ideas / Feature Requests]
Could be important to extend BTCPayserver support to networks such as i2p?

Currently, there are more frequent problems in Tor, and many DDoS attacks and I think i2p could be an interesting alternative.
Bitcoin Core already uses it to connect to other nodes and p2p exchanges like Robosats have implemented it successfully.

I know Nostr is also mentioned now, however, I think Nostr is still very much in its infancy. I2p is already a reality, although little used.

Unfortunately, I'm not a tech guy and I don't know how to implement this. I know that is possible to create a tunnel (like a Tor hidden service), but when I try it, I have a message from BTCPayserver that says that is not possible to login because the connection is not an SSL connection or Tor connection.

**Comment by NicolasDorier** [comment]
Should be more related to https://github.com/btcpayserver/btcpayserver-docker
Why not

**Comment by decentralizedb** [comment]
Thank you for your answer, Nicolas.
With docker, the flexibility to deploy is high. But in my opinion (and without being a technical expert), you lose some possibility to configure the BTCPayserver if you don't have some specialised experience (without using docker).
For example, I have a vanilla installation, and I don't use Nginx, but apache2, it's just an example. 
So, I think that if it could be possible to develop it in both ways, an installation under docker and manual could be, in my opinion, the ideal.
